extensions/wikia/WikiaWhiteList/WikiaWhiteList.php

```diff
@@ -52,7 +53,7 @@ function wfWhiteListParse($text) {
 	//---
 	if (preg_match(sprintf(REGEX_PARSE_WHITELIST,'rel','nofollow'), $text)) {
 		//---
-		preg_match('#href\s*?=\s*?[\'"]?([^\'" ]*)[\'"]?#i', $text, $captures);
+		preg_match(HREF_PARSE_WHITELIST, $text, $captures);
```

Comment:
I think adding a `LinkerMakeExternalLink` listener instead of putting this in the `WikiaWhiteList` is cleaner. `WikiaWhiteList` has nothing to do with using primary wiki domains in external links.